engine.php

```diff
@@ -44,7 +49,7 @@
                             'end' => $phpcs_issue_data['line']
                         )
                     ),
-                    'remediation_points' => $phpcs_issue_data['severity'] * 75000
+                    'remediation_points' => Sniffs::pointsFor($source) || $phpcs_issue_data['severity'] * 75000
```

Comment:
What's the precedence of the operators here? Is the 75000 applied to both options or only the `phpcs_issue_data` one?